happytransformer/fine_tuning_util.py

```diff
@@ -0,0 +1,58 @@
+"""
+Contains functions that are shared amongst the children of the HappyTrainer class.
+
+Based on
+https://github.com/huggingface/transformers/blob/master/examples/pytorch/language-modeling/run_mlm.py
+and
+https://github.com/huggingface/transformers/blob/master/examples/pytorch/language-modeling/run_clm.py
+"""
+
+
+def preprocess_concatenate(tokenizer, datasets, args, mlm=True):
+    """
+    :param tokenizer: tokenizer for a transformer model
+    :param datasets: a datasets.Dataset object
```

Comment:
datasets is a dataset object? that seems weird

Reply:
33211f7

Reply:
I renamed it to "dataset." I also applied this same change to other sections of the codebase.